README.md

```diff
@@ -36,7 +36,29 @@
 
 A detailed changelog is available in the [releases](https://github.com/angular-schule/angular-cli-ghpages/releases) section.
 
-**GitHub Actions** is now a first citizen alongside Travis CI and CircleCi. The token `GITHUB_TOKEN` is now supported. Learn everything you need to know in the following article.
+**⚠️ BREAKING CHANGE (v1)**
+
+Starting with version 1 the option `--configuration` was renamed to `--build-target`.
+
+BEFORE (_does not work_):
+
+```
+ng deploy --configuration=test
+```
+
+NOW:
+
+```
+ng deploy --build-target=test
+```
+
+If you use the old syntax, you will probably receive the following error:
+
+> An unhandled exception occurred: Configuration 'test' is not set in the workspace.
+
+<br>
+
+**🐙 GitHub Actions** is now a first citizen alongside Travis CI and CircleCi. The token `GITHUB_TOKEN` is now supported. Learn everything you need to know in the following article.
```

Comment:
```suggestion
**🐙 GitHub Actions** is now properly supported alongside Travis CI and CircleCi. The token `GITHUB_TOKEN` is also supported. Learn everything you need to know in the following article.
```

The Google guide for inclusive documentation discourages the use of "first-class citizen": https://developers.google.com/style/inclusive-documentation#features-and-users

Reply:
Good to know! 👍